.ci/scripts/util.sh

```diff
@@ -2,65 +2,129 @@
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 
-metrics_collection_start ()
+# Start the metrics collection of both docker stats and the miscellaneous os statistics.
+# Parameters:
+#     container_info:  the name of the docker file from which to gather statistics
+#     sample_time:  The time between sampling of the statistics.
+metrics_collection_start()
 {
-    # container name should always contain pge_docker_tag, PGE, and runconfig (at a minimum)
-    container_name=$1
+    local pge=$1
+    # Split seconds argument into docker name and tag
+    local cont_name="$(echo "$2" | cut -d':' -f1)"
+    local container_tag="$(echo "$2" | cut -d':' -f2)"
+    # cont_info should always contain pge_docker_tag, PGE, and runconfig (at a minimum)
+    # container_info="${pge}_${cont_name}_${container_tag}"
 
+    local container_info="$pge"
+
+    # If no sample_time value is passed - default to a value of 1
+    if [[ -z "$2" ]]
```

Comment:
Should this be `if [[ -z "$3" ]]` now that there's an additional input argument?

Reply:
Thanks.  Good catch.